arcgis-ios-sdk-samples/Scenes/Surface placements/SurfacePlacements.storyboard

```diff
@@ -1,35 +1,106 @@
+                                                <constraint firstItem="lcW-2W-3Yg" firstAttribute="leading" secondItem="Pi5-ud-tAa" secondAttribute="leading" id="3zF-XW-5lI"/>
+                                                <constraint firstAttribute="trailing" secondItem="lcW-2W-3Yg" secondAttribute="trailing" constant="8" id="COp-3Z-crH"/>
+                                            </constraints>
+                                            <viewLayoutGuide key="safeArea" id="Pi5-ud-tAa"/>
+                                        </view>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" id="L5g-QB-jPk">
+                                        <slider key="customView" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" value="70" minValue="0.0" maxValue="140" id="1gl-rZ-oz2">
+                                            <rect key="frame" x="187" y="10" width="140" height="34"/>
+                                            <autoresizingMask key="autoresizingMask" flexibleMinX="YES" widthSizable="YES" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                            <connections>
+                                                <action selector="sliderValueChanged:" destination="Wm5-sQ-VeD" eventType="valueChanged" id="CLi-Ie-FB4"/>
+                                            </connections>
+                                        </slider>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="wna-Tk-U5n"/>
+                                </items>
+                            </toolbar>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="eWj-fZ-V3J">
+                                <rect key="frame" x="0.0" y="769" width="414" height="44"/>
+                                <items>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="PVB-DB-c7R"/>
+                                    <barButtonItem style="plain" id="4bj-UX-pB2">
+                                        <segmentedControl key="customView" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="top" segmentControlStyle="bar" selectedSegmentIndex="0" id="yTQ-cO-QvB">
+                                            <rect key="frame" x="20" y="5.5" width="374" height="33"/>
+                                            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                            <segments>
+                                                <segment title="Draped Billboarded"/>
```

Comment:
What you have is fine, but since both segments have the word "Draped", you might consider moving that outside the control into a label for the control.

Reply:
Yes this makes sense. When I changed it I wasn't sure whether to keep it outside segmented control, so I copied dotnet's approach: https://github.com/Esri/arcgis-runtime-samples-dotnet/pull/806/files?short_path=d1ee392#diff-d1ee392254bb7519df0497bfbb792d78